app/code/local/LoyaltyLion/CouponImport/controllers/Adminhtml/QuickSetupController.php

```diff
@@ -0,0 +1,212 @@
+        //Allow everything
+        $resources = array(
+            Mage_Api2_Model_Acl_Global_Rule::RESOURCE_ALL => array(
+                null => Mage_Api2_Model_Acl_Global_Rule_Permission::TYPE_ALLOW
+            )
+        );
+
+        $id = $role->getId();
+        foreach ($resources as $resourceId => $privileges) {
+            foreach ($privileges as $privilege => $allow) {
+                if (!$allow) {
+                    continue;
+                }
+
+                $rule->setId(null)
+                    ->isObjectNew(true);
+
+                $rule->setRoleId($id)
+                    ->setResourceId($resourceId)
+                    ->setPrivilege($privilege)
+                    ->save();
+            }
+        }
+
+        return $roleId;
+    }
+
+    public function assignToRole($userId, $roleId) {
+        Mage::log("[LoyaltyLion] Assigning current admin user to LoyaltyLion REST role");
+        $model = Mage::getResourceModel('api2/acl_global_role');
+        $model->saveAdminToRoleRelation($userId, $roleId);
+    }
+
+    public function enableAllAttributes() {
+        Mage::log("[LoyaltyLion] Enabling attribute access for this REST role");
+        $type = 'admin';
+        $ruleTree = Mage::getSingleton(
+            'api2/acl_global_rule_tree',
+            array('type' => Mage_Api2_Model_Acl_Global_Rule_Tree::TYPE_ATTRIBUTE)
+        );
+        /** @var $attribute Mage_Api2_Model_Acl_Filter_Attribute */
+        $attribute = Mage::getModel('api2/acl_filter_attribute');
+        /** @var $collection Mage_Api2_Model_Resource_Acl_Filter_Attribute_Collection */
+        $collection = $attribute->getCollection();
+        $collection->addFilterByUserType($type);
+        /** @var $model Mage_Api2_Model_Acl_Filter_Attribute */
+        foreach ($collection as $model) {
+            $model->delete();
+        }
+        $resources = array(
+            Mage_Api2_Model_Acl_Global_Rule::RESOURCE_ALL => array(
+                null => Mage_Api2_Model_Acl_Global_Rule_Permission::TYPE_ALLOW 
+            )
+        );
+        foreach ($resources as $resourceId => $operations) {
+            if (Mage_Api2_Model_Acl_Global_Rule::RESOURCE_ALL === $resourceId) {
+                $attribute->setUserType($type)
+                    ->setResourceId($resourceId)
+                    ->save();
+            } else {
+                foreach ($operations as $operation => $attributes) {
+                    $attribute->setId(null)
+                        ->isObjectNew(true);
+                    $attribute->setUserType($type)
+                        ->setResourceId($resourceId)
+                        ->setOperation($operation)
+                        ->setAllowedAttributes(implode(',', array_keys($attributes)))
+                        ->save();
+                }
+            }
+        }
+    }
+
+    public function generateOAuthCredentials($name, $userID) {
+        Mage::log("[LoyaltyLion] Generating OAuth credentials");
```

Comment:
If they have logging enabled, it goes to MAGE_DIR/var/log/system.log by default.